service-commands/scripts/setup-instance.sh

```diff
@@ -95,14 +98,17 @@ case "$service" in
 		;;
 	remote-dev)
 		wait_for_instance $provider $user $name
-		execute_with_ssh $provider $user $name \
-			"[[ ! -d ~/audius-protocol ]]" \
-			"&& git clone https://github.com/AudiusProject/audius-protocol.git" \
-			"&& yes | bash audius-protocol/service-commands/scripts/provision-dev-env.sh" \
-			"&& yes | bash audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
+		echo "Waiting for instance $name on $provider to be ready for ssh connections... You may see some SSHc connection errors during this time but instance should eventually come up."
+		if [ "${fast:-0}" -eq "0" ]; then
+			execute_with_ssh $provider $user $name \
+				"[[ ! -d ~/audius-protocol ]]" \
+				"&& git clone https://github.com/AudiusProject/audius-protocol.git" \
+				"&& yes | bash audius-protocol/service-commands/scripts/provision-dev-env.sh" \
+				"&& yes | bash audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
```

Comment:
Move 107 to before 106 and use `PROTOCOL_DIR=/home/ubuntu/audius-protocol bash...`.

But `PROTOCOL_DIR=audius-protocol bash...` may also work! I have the full path since I was hitting that issue with `audius_client` != `audius-client`.

Reply:
I have confirmed that `PROTOCOL_DIR=audius-protocol bash...` works

Reply:
Gotcha - I ended up calling set-git-refs.sh from inside provision-dev-env and passing the git refs arguments to provision-dev-env in a little refactor - take a look! Testing this now.

Reply:
Ah nice! That looks good! I would only suggest moving the `set-git-refs.sh` as the first line in `setup_audius_repos()` so that way the install commands will all use the target tag.

Reply:
makes sense!